### ISSUE TITLE
applying landmask as documentation says brake dimensions ordering

### ISSUE THREAD

**Issue by oarcher** [umr-lops/xsar#11]
in example notebook [mask.ipynb](blob/develop/docs/examples/mask.ipynb), mask is applied with the following code:

```python
 ds['sigma0_ocean'] = xr.where(ds['land_mask'], np.nan, ds['sigma0'])
```

however, if `ds['sigma0']` shape is `(pol,atrack,xtrack)` ,  `ds['sigma0_ocean']` will have shape `(atrack,xtrack,pol)`

It shouldn't be harmful, but this can break not well written further code. 

So for good practice, we should provide an example that do not brake dimension ordering. Something like:

```python
xr.where(ds['land_mask'], np.nan, ds['sigma0']).transpose(*ds['sigma0'].dims)
```

or 

```python
xr.concat( [ xr.where(ds['land_mask'], np.nan, ds['sigma0'].sel(pol=p)) for p in ds['sigma0'].pol], dim='pol')
```

**Comment by oarcher** [comment]
closed by PR #12

**Comment by oarcher** [comment]
closed by PR #12
